### DISCUSSION TITLE
Source & Style Homework

### DISCUSSION THREAD

**Discussion by ZoeLeBlanc** [CultureAsData-UIUC/is310-fall-2024#4] [Assignments]
Please add a link below to your Source & Style Homework [https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/07-intro-html#homework-source-and-style](https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/07-intro-html#homework-source-and-style). You should post a link to your `source-and-style` folder. Remember this assignment has two parts: inspect a website and creating your own HTML page.

**Comment by haydnelm** [comment]
https://github.com/haydnelm/is310-coding-assignments/tree/main/source-and-style

**Comment by tinayang922** [comment]
https://github.com/tinayang922/is310-coding-assignments/tree/main/source-and-style

**Comment by yosefstifanos** [comment]
https://github.com/yosefstifanos/is310-coding-assignments/tree/main/source-and-style

**Comment by Cindy1226** [comment]
https://github.com/Cindy1226/is310-coding-assignments/tree/main/source-and-style

**Comment by GloriaLiu930** [comment]
https://github.com/GloriaLiu930/is310-coding-assignments/tree/main/source-and-style

**Comment by averyWebster** [comment]
https://github.com/averyWebster/is310-coding-assignments/tree/main/source-and-style

**Comment by kohtas2** [comment]
https://github.com/kohtas2/is310-coding-assignments/tree/main/source-and-style

**Comment by ahmadsawwan** [comment]
https://github.com/ahmadsawwan/ahmadsawwan-is310-coding-assignments/tree/main/source-and-style

**Reply by ZoeLeBlanc** [reply]
This is great @ahmadsawwan but just FYI your images are not showing up in your `README.md`. If it's helpful the syntax for images in Markdown is `![Alt Text](file path to the image)`. So in your case it might be `![metadata](images/screenshot.png)`. Overall good work though 👍

**Reply by ahmadsawwan** [reply]
I think it should be fixed now!

**Comment by frangara** [comment]
https://github.com/frangara/is310-coding-assignments/tree/main/source-and-style

**Comment by jasonmorris8** [comment]
https://github.com/jasonmorris8/is310-coding-assignments/tree/main/source-and-style

**Comment by shoirathi** [comment]
https://github.com/shoirathi/is310-coding-assignments/tree/main/source-and-style

**Reply by ZoeLeBlanc** [reply]
This is great @shoirathi  but just FYI your images are not showing up in your `README.md`. If it's helpful the syntax for images in Markdown is `![Alt Text](file path to the image)`. So in your case it might be `![metadata](images/screenshot.png)`. Overall good work though 👍

**Comment by JBrown-0** [comment]
https://github.com/JBrown-0/is310-coding-assignments/blob/main/index.html

**Reply by ZoeLeBlanc** [reply]
Hi @JBrown-0 ! This is a great start but there's a few issues I'm seeing. I would be happy to help you after class today or feel free to set up a calendly meeting with me [https://calendly.com/zleblanc/chat-with-prof-zoe]( https://calendly.com/zleblanc/chat-with-prof-zoe)

1. First, it looks like your local git is misconfigured and that you don't have the correct user name in your local git so it is not associating your commits with GitHub user profile. You can follow the steps here to fix it [https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/01-course-tools/#git-configuration](https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/01-course-tools/#git-configuration), though also happy to help with this one. Also looks like I mentioned this to you previously just fyi [https://github.com/CultureAsData-UIUC/is310-fall-2024/discussions/1#discussioncomment-10604420](https://github.com/CultureAsData-UIUC/is310-fall-2024/discussions/1#discussioncomment-10604420)
2. It seems like you're missing some of the folders you should have so far, both for this assignment and previous ones.
3. Your `index.html` contains what looks like git errors in front of the HTML which will make it tough for a browser to render correctly.

Again happy to help you resolve these and this is a good start so please don't hesitate to reach out if that would be helpful 👍

**Comment by OctaneCodes** [comment]
https://github.com/OctaneCodes/is310-coding-assignments/tree/main/source-and-style

**Reply by ZoeLeBlanc** [reply]
Hi @OctaneCodes ! This looks great except your `#` symbols are not correctly rendering in your `README.md` because there is no space between the symbol and the text. It's a small thing but wanted to let you know. Overall though good work 👍

**Reply by OctaneCodes** [reply]
Fixed it.

**Comment by yunchaecho2** [comment]
https://github.com/yunchaecho2/is310-coding-assignments/tree/main/source-and-style

**Reply by ZoeLeBlanc** [reply]
Hi @yunchaecho2 ! This looks good overall but I noticed two issues. The first is that you have a `.DS_Store` file in your folder and also that your commits are not being associated with your GitHub user profile.

To fix the first issue, you should follow these steps:

1. Create a `.gitignore` locally in your `is310-coding-assignments` folder
2. Add `.DS_Store` to the top of that file and save it.
3. Delete your local `.DS_Store` file locally. You can do that with the following command, though be sure to run this in your `is310-coding-assignments` folder. The command would be `rm -rf .DS_Store` though you will need to do it in each folder likely.
4. Then each folder where you delete the file type `git rm --cached .DS_Store` in your terminal. It's ok if it shows an error, we just want to remove the file from your git history.
5. Do your normal `git add .`, `git commit -m "deleted .DS_Store and added .gitignore", and finally `git push origin main`. 

To fix the second issue, you should follow the steps here to fix it [https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/01-course-tools/#git-configuration](https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/01-course-tools/#git-configuration), though also happy to help with this one.

 Overall though good work 👍

**Comment by lhayedimarucut** [comment]
https://github.com/lhayedimarucut/is310-coding-assignments/tree/main/source-and-style

**Reply by ZoeLeBlanc** [reply]
Hi @lhayedimarucut ! This looks great overall but I noticed one issue for you to fix, specifically you have a `.DS_Store` file in your repository.

To fix it, please follow these steps and feel free to reach out if you have questions:

1. Create a `.gitignore` locally in your `is310-coding-assignments` folder
2. Add `.DS_Store` to the top of that file and save it.
3. Delete your local `.DS_Store` file locally. You can do that with the following command, though be sure to run this in your `is310-coding-assignments` folder. The command would be `rm -rf .DS_Store` though you will need to do it in each folder likely.
4. Then each folder where you delete the file type `git rm --cached .DS_Store` in your terminal. It's ok if it shows an error, we just want to remove the file from your git history.
5. Do your normal `git add .`, `git commit -m "deleted .DS_Store and added .gitignore", and finally `git push origin main`. 

 Overall though good work 👍

**Comment by divyaj5** [comment]
https://github.com/divyaj5/is310-coding-assignments/tree/master/source-and-style

**Comment by Rebe-Rod** [comment]
https://github.com/Rebe-Rod/is310-coding-assingments/tree/main/source-and-style

**Comment by LucasWashor** [comment]
https://github.com/LucasWashor/is310-coding-assignments/tree/main/source-and-style

**Comment by henRotunno** [comment]
https://github.com/henRotunno/is310-coding-assignments/tree/main/source-and-style

**Reply by ZoeLeBlanc** [reply]
Hi @henRotunno ! This is great work and your `index.html` looks great. One small issue is your `inspect` file. This file should be the `README.md` file instead. I would recommend creating a `README.md`, placing the text in the `inspect` file, and then you can also embed your images. Happy to help if you run into more git issues, but overall good job 👍

**Reply by henRotunno** [reply]
Thank you for the feedback, I have changed the inspect file and created a README.md with the same information: https://github.com/henRotunno/is310-coding-assignments/tree/main/source-and-style

**Comment by AG-Samaniego** [comment]
https://github.com/AG-Samaniego/is310-coding-assignments/tree/main/source-and-style
This is partially finished (my bad) I'm completing it today rip

**Reply by ZoeLeBlanc** [reply]
Thanks for sharing @AG-Samaniego ! You're very close so feel free to ping me here when you finish the assignment fully 👍

**Comment by AlanK3** [comment]
https://github.com/AlanK3/is310-coding-assignments/tree/main/source-and-style

**Comment by officalcharles** [comment]
https://github.com/officalcharles/is310-coding-assignments/tree/master/source-and-style

**Comment by ryanpearlman1** [comment]
https://github.com/ryanpearlman1/is310-coding-assignments/tree/main/source-and-style

**Comment by SamuelSokolovsky** [comment]
https://github.com/SamuelSokolovsky/is310-coding-assignments/tree/main/source-and-style

**Comment by elee33** [comment]
https://github.com/elee33/is310-coding-assignments/tree/master/source-and-style

**Comment by theajk2** [comment]
https://github.com/theajk2/is310-coding-assignments/tree/main/source-and-style

**Comment by adg77** [comment]
https://github.com/adg77/is310-coding-assignments/tree/main/source-and-style

**Comment by yukthas2** [comment]
https://github.com/yukthas2/is310-coding-assignments/tree/main/source-and-style

**Reply by ZoeLeBlanc** [reply]
Just FYI @yukthas2 that your `README.md` is not quite rendering properly with the `#` symbols. They require a space between the symbol and the text to render. But overall good work 👍

**Comment by esr27** [comment]
https://github.com/esr27/is310-coding-assignments/tree/main/source-and-style

**Comment by chuzesta** [comment]
https://github.com/chuzesta/is310-coding-assignments/tree/main/source-and-style 
*forgot to submit
